### PR TITLE
document use of asb export-bitcoin-wallet

### PR DESCRIPTION
**Pull request by icy-ux**
Add to the `asb` document @delta1's text on exporting the Bitcoin wallet
descriptor and importing the descriptor into Sparrow Wallet. Addresses #878

TODO: maybe it would make sense to add the images to Git, instead of
linking to files on Github?

### CODE REVIEW

**Comment by icy-ux** [comment]
Looks like one integration test fails, but it's not my fault

File: docs/asb/README.md [hunk 1]
```diff
@@ -161,3 +161,45 @@ May 01 01:32:05.018  INFO Tor found. Setting up hidden service.
...
+{"descriptor":"wpkh(tprv8Zgredacted.../84'/1'/0'/0/*)","blockheight":2415616,"label":"asb-testnet"}
+```
+
+The wallet can theoretically be directly imported into
+[bdk-cli](https://bitcoindevkit.org/bdk-cli/installation/) but it is easier to
+use Sparrow Wallet.
+
+Sparrow wallet import works as follows:
+
+    * File -> New wallet -> Give it a name
```
Comment (binarybaron) [inline docs/asb/README.md:181]: These bulletpoints are formatted in a very weird way. Please use normal bulletpoints.

**Comment by binarybaron** [comment]
> Looks like one integration test fails, but it's not my fault

It's a timeout, happens from time to time

**Comment by binarybaron** [comment]
> Looks like one integration test fails, but it's not my fault

It's a timeout, happens from time to time

**Comment by icy-ux** [comment]
>These bulletpoints are formatted in a very weird way. Please use normal bulletpoints.

Fixed. The formatting was directly copied from @delta1's original text,
so I assumed it was formatted according the project style.

**Comment by delta1** [comment]
Thanks @icy-ux, I think this could be written better and not need the images, and I'd also like to figure out how to import it into the `bdk cli` correctly.

**Comment by binarybaron** [comment]
> >These bulletpoints are formatted in a very weird way. Please use normal bulletpoints.
> 
> Fixed. The formatting was directly copied from @delta1's original text,
> so I assumed it was formatted according the project style.
> 

Yeah true, I see why you did that. Really no offense intended. Thank you alot for the PR, we really need to improve the documentation overall.

**Comment by icy-ux** [comment]
>@icy-ux please run dprint fmt and then I’m happy for this to go in, it can always be updated later.

Done.

Unfortunately the backticks in my commit message were cut out along the
way, so it's not the best commit message in the world.

**Comment by delta1** [comment]
Thanks! No problem, please squash your commits into 1, make sure it’s GPG signed, force push and then we’ll get this merged. 

https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

**Review by binarybaron — APPROVED** [review]

**Comment by delta1** [comment]
@ikmckenz @icy-ux please add a commit including the images and updating the paths in the md file

**Comment by icy-ux** [comment]
@delta1 done, should be ready to merge

**Review by ikmckenz — APPROVED** [review]

**Pull request merged**